components/AdditionalResources.tsx

```diff
@@ -8,36 +9,31 @@ const AdditionalResources: React.FC = () => {
         ADDITIONAL RESOURCES
       </h1>
       <hr className="my-2" />
-      <a
-        rel="noopener noreferrer"
-        target="_blank"
-        className="mx-3 "
-        href="https://www.notion.so/Setup-Instructions-fc8f8fcc1376482ead839fa6b1034cb4"
-      >
-        Setup Instructions
-      </a>
+      <NavLink
+        text="Setup Instructions"
+        path="https://www.notion.so/Setup-Instructions-fc8f8fcc1376482ead839fa6b1034cb4"
+        className="mx-3"
+        external
+        blank
+      />
       <hr className="my-2" />
-      <a className="mx-3 " href="/">
-        Github Basics
-      </a>
+      <NavLink text="Github Basics" path="#" className="mx-3" />
       <hr className="my-2" />
-      <a
+      <NavLink
+        text="Stack Overflow"
+        path="https://stackoverflow.com/"
         className="mx-3"
-        href="https://stackoverflow.com/"
-        target="_blank"
-        rel="noopener noreferrer"
-      >
-        Stack Overflow
-      </a>
+        external
+        blank
```

Comment:
is it safe to say that its always gonna be `blank` when link is external? I can't think of a scenario where we want users to leave our page....

Reply:
fixed!